acceptance/analyzer_test.go

```diff
@@ -41,22 +40,21 @@ var (
 	analyzerPath         = "/cnb/lifecycle/analyzer"
 	cacheFixtureDir      = filepath.Join("testdata", "analyzer", "cache-dir")
 	daemonOS, daemonArch string
-	readOnlyRegistry     *ih.DockerRegistry
-	authRegistry         *ih.DockerRegistry
-	customRegistry       *ih.DockerRegistry
+	testRegistry         *ih.DockerRegistry
 	registryNetwork      string
 	fixtures             analyzeFixtures
 )
 
 type analyzeFixtures struct {
-	authRegAppImage       string
+	authRegAppImage       string // read and write access
 	authRegAuthConfig     string
-	authRegCacheImage     string
-	authRegOtherAppImage  string
+	authRegCacheImage     string // read-only
```

Comment:
Is this an accurate comment? Don't we need read/write for cache?

Reply:
Mmm, you are right, it shouldn't be read-only. I will fix it

Reply:
Fixed in the following [commit](https://github.com/buildpacks/lifecycle/pull/685/commits/27c678fc98ac3d25a2f3c0727ebff90aa0640ba1)